be/src/olap/rowset/segment_group.cpp

```diff
@@ -146,35 +146,16 @@ SegmentGroup::~SegmentGroup() {
     _seg_pb_map.clear();
 }
 
```

Comment:
There is a separate pending delta folder in the past. We should remove the folder?

Reply:
OK, I will remove it in SegmentGroup::remove_old_files